src/main/kotlin/com/dto/StartedReservationDto.kt

```diff
@@ -0,0 +1,16 @@
+package com.dto
+
+import com.model.DriverType
+import java.time.LocalDateTime
+
+class StartedReservationDto(
+        val id: Long,
+
+        val driverType: DriverType,
+
+        val carLicensePlate: String,
+
+        val startTime: LocalDateTime,
+
+        val currencyCode: String
```

Comment:
czemu nie jakiś enum?

Reply:
w założeniu użyłbym "Java Money and the Currency API" która przyjmuje stringi, aczkolwiek enumy też mogą mieć przecież wartości, więc z perspektywy zgodzę się że enumy byłyby lepsze.